src/main/java/bio/terra/service/snapshotbuilder/query/TrueLiteral.java

```diff
@@ -0,0 +1,11 @@
+package bio.terra.service.snapshotbuilder.query;
+
+public class TrueLiteral implements SqlExpression {
+
+  @Override
+  public String renderSQL(SqlRenderContext context) {
```

Comment:
I don't know if this is too cloud specific, but thoughts on a `BooleanLiteral` class? 

```
public class BooleanLiteral implements SqlExpression {

 private String azureBooleanLiteral;
 private String gcpBooleanLiteral;
 
 public BooleanLiteral(boolean booleanLiteral) {
   if (booleanLiteral) {
      this.azureBooleanLiteral = "1";
      this.gcpBooleanLiteral = "TRUE";
    } else {
     this.azureBooleanLiteral = "0";
     this.gcpBooleanLiteral = "FALSE;
   }
}

 @Override
  public String renderSQL(SqlRenderContext context) {
      return context.getPlatform().choose(this.gcpBooleanLiteral, this.azureBooleanLiteral);
  }
}



}
```

Reply:
That's a good point, it'd be better to have a more general solution here. Instead of doing this, I modified `Literal` to support boolean types this way.